### DISCUSSION TITLE
The PowerCLI Command New-VM delivers with vcsim the error: The valid range for VM memory size is between 4 MB and 0 MB

### DISCUSSION THREAD

**Discussion by StSchnell** [vmware/govmomi#3343] [Community Help]
I installed the current version from vcsim, PowerShell and PowerCLI. I start vcsim with username, password and tls=false arguments. The connection with the PowerCLI to the simulated vCenter works fine. The cmdlets Get-Datacenter, Get-VM, Get-VMHost and Get-Datastore delivers all the expected results. Everything looks very good, but when I try to create a VM with the New-VM cmdlet an error message occurs.
E.g. the code ...
```powershell
$cluster = Get-Cluster -Name DC0_C0
New-VM -Name vm-test -ResourcePool $cluster
```
... delivers `New-VM: 1/22/2024 10:47:12 AM New-VM The valid range for VM memory size is between 4 MB and 0 MB.`
Also I tried it with -MemoryGB and other different variants, but I get always the same result.
What is my mistake? Does anyone have a tip or an idea?
Thanks and best regards
Stefan

**Comment by mlytle4218** [comment]
I'm having a similar issue. Is the approach wrong?
